rust-toolchain.toml

```diff
@@ -1,2 +1,2 @@
 [toolchain]
-channel = "1.60.0"
+channel = "nightly"
```

Comment:
So the crate couldn't compile:
```
error: package `hashbrown v0.14.0` cannot be built because it requires rustc 1.64.0 or newer, while the currently active rustc version is 1.60.0

```
And I see the CI compiles with the nightly toolchain, so I assume it is safe to change.
Please let me know this is not true.